Phase2L1Trigger/CalibratedDigis/README.md

```diff
@@ -0,0 +1,2 @@
+# CalibratedDigis
```

Comment:
There is no package `Phase2L1Trigger/CalibratedDigis`.
Please move this into a package to `L1Trigger/DTPhase2Trigger`

same goes for the rest of the files.

Overall comment: 
Please rename this new package `L1Trigger/DTPhase2Trigger` to `L1Trigger/DTTriggerPhase2`